core/src/main/java/hudson/util/AtomicFileWriter.java

```diff
@@ -108,7 +107,7 @@ public AtomicFileWriter(@Nonnull Path destinationPath, @Nonnull Charset charset)
             throw new IOException("Failed to create a temporary file in "+ dir,e);
         }
 
-        core = Files.newBufferedWriter(tmpPath, charset, StandardOpenOption.SYNC);
+        core = Files.newBufferedWriter(tmpPath, charset);
```

Comment:
This change means that the referenced JIRA is not fixed.

Reply:
@batmat I'm happy to take another look once SYNC is restored, was this the only failing unit test? https://github.com/jenkinsci/jenkins/pull/2548/commits